### DISCUSSION TITLE
Harvester Id not showing up

### DISCUSSION THREAD

**Discussion by Alan911Black** [Chia-Network/chia-blockchain#7799] [Support]
Not sure if it is a bug, but I have some issues when farming to a pool. I have 3 Harvesters, but only two are showing up in the Harvester Id list. I also think I am not getting Partials, on pool.space my network space is way off. 

how can I check what is my Harvester Id? none of the commends are giving me the  Harvester Id
I like to match each computer, to find out what Harvester is not included
thanks

**Comment by archerallstars** [comment]
It's not a bug. If you have a high difficulty point, you would not send the partials as much, thus the harvesters with the least plots might not show up in the pool at all unless all your harvesters have a similar number of plots.

However, all the harvesters will show up eventually (the one with the least plot will pop in and out which is normal).

**Comment by Alan911Black** [comment]
i have a server that host all the plots and two computers that creates plots. as soon as i move them to the server, it seams i am loosing the plots. how can i fix that?
do you know how to find out the Harvester Id?

**Reply by archerallstars** [reply]
> i have a server that host all the plots and two computers that creates plots. as soon as i move them to the server, it seams i am loosing the plots. how can i fix that?
> do you know how to find out the Harvester Id?

I don't think you will lose any plots if you successfully moved them to the server machine. Those plots will be farmed under the server's harvester instead of the plotters' harvester.

I don't know any CLI to get the harvester ID either. It seems the ID can only be found under the GUI. I also asked this in [another thread](https://github.com/Chia-Network/chia-blockchain/discussions/7800).

**Reply by Alan911Black** [reply]
yes, you are correct. i wish it worked like that.
i will post it there also
